code/modules/mob/living/blood.dm

```diff
@@ -6,55 +6,166 @@
 				BLOOD SYSTEM
 ****************************************************/
 
+/// Returns whether this mob can have blood.
+/// Use the CAN_HAVE_BLOOD(mob) macro instead.
+/// Unless you're the one updating blood status.
+/mob/living/proc/can_have_blood()
+	return get_default_blood_volume() > 0
+
+/mob/living/carbon/can_have_blood()
+	return !HAS_TRAIT(src, TRAIT_NOBLOOD)
+
+/// Returns the default blood volume of this mob. Useful for healing bloodloss.
+/mob/living/proc/get_default_blood_volume()
+	return initial(blood_volume)
+
+/mob/living/carbon/get_default_blood_volume()
+	return BLOOD_VOLUME_NORMAL
+
+/// Gets the base blood volume of the mob, before scalars like Saline-Glucose Solution are applied.
+/// For effects like oxyloss damage from blood volume, use [proc/get_modified_blood_volume] instead.
+/mob/living/proc/get_blood_volume()
+	return CAN_HAVE_BLOOD(src) ? blood_volume : 0
```

Comment:
Why do we have a getter for `get_blood_volume`? 
It seems like the only behavior is to return 0 if they mob has no blood but -